src/main/java/phase/PhaseUtils.java

```diff
@@ -0,0 +1,45 @@
+package phase;
+
+import at.ac.tuwien.ifs.sge.game.risk.board.Risk;
+import at.ac.tuwien.ifs.sge.game.risk.board.RiskBoard;
+
+public class PhaseUtils {
+    public static boolean stillUnoccupiedTerritories(RiskBoard board) {
+        return board.getTerritories().values().stream().anyMatch(riskTerritory -> riskTerritory.getTroops() == 0);
+    }
+
+    public static boolean initialPlacementFinished(RiskBoard board, int playerNumber, int numberOfStartingTroops) {
```

Comment:
stupid question but why is this not the same as !stillUnoccupiedTerritories ?

Reply:
Should be the same, I just did not think of it